client/app/google-services.json

```diff
@@ -0,0 +1,39 @@
+{
+  "project_info": {
+    "project_number": "275792153598",
+    "project_id": "interval-acc57",
+    "storage_bucket": "interval-acc57.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:275792153598:android:9ced350d5e42e58fd00eaf",
+        "android_client_info": {
+          "package_name": "com.example.interval"
```

Comment:
incorrect package name

Reply:
package_name is right. This one is used for the client